pkg/tests/is_compressed_test.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests if input files are compressed."""
+
+import unittest
+
+from bazel_tools.tools.python.runfiles import runfiles
+
+
+class IsCompressedTest(unittest.TestCase):
+
+  def get_file_under_test(self, file_name):
+    """Get the file path to a generated zip in the runfiles."""
+
+    return self.data_files.Rlocation(
+        "rules_pkg/tests/" + file_name
```

Comment:
Consider `os.environ['WORKSPACE_NAME']` instead of `rules_pkg` directly for portability.